### PR TITLE
Publish releases to maven central

### PR DESCRIPTION
**Pull request by duncte123**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        required: false
+      ORG_GRADLE_PROJECT_signingInMemoryKeyPassword:
+        required: false
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      MAVEN_USERNAME: ${{ secrets.M2_USER }}
+      MAVEN_PASSWORD: ${{ secrets.M2_PASS }}
```
Comment (topi314) [inline .github/workflows/build.yml:26]: you can use our existing user & password her

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      -   name: Checkout
+          uses: actions/checkout@v3
+          with:
+            fetch-depth: 0
+
+      -   name: Setup Java
+          uses: actions/setup-java@v3
+          with:
+            distribution: zulu
+            java-version: 11
```
Comment (topi314) [inline .github/workflows/build.yml:41]: shouldn't this be java 17?

File: build.gradle.kts [hunk 3]
```diff
@@ -156,12 +165,46 @@ publishing {
...
-fun getSuffix(): String {
-    if (isSnapshot) {
-        return "-SNAPSHOT_${System.getenv("GITHUB_RUN_NUMBER")}"
-    }
+afterEvaluate {
+    plugins.withId("com.vanniktech.maven.publish.base") {
+        configure<MavenPublishBaseExtension> {
+            coordinates(group.toString(), project.the<BasePluginExtension>().archivesName.get(), version.toString())
+
+            if (System.getenv("MAVEN_USERNAME") != null && System.getenv("MAVEN_PASSWORD") != null) {
```
Comment (topi314) [inline build.gradle.kts:173]: MAVEN_USERNAME & MAVEN_PASSWORD are for https://maven.arbjerg.dev
ideally we would publish to both repos, not sure what this check is for tbh

**Review by topi314 — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (duncte123) [reply .github/workflows/build.yml]: Yes it should

File: build.gradle.kts (hunk 3 above)
Comment (duncte123) [reply build.gradle.kts:173]: me neither, I copied it from lavaplayer

**Review by topi314 — APPROVED** [review]

**Pull request merged**